packages/terra-framework-docs/src/terra-dev-site/doc/data-grid/FlowsheetDataGrid.4/About.5.doc.mdx

```diff
@@ -121,6 +133,8 @@ A column specifies the data to render a cell in the header row of the Flowsheet
 |**displayName**|string|optional|none|String of text to render in the column header cell.|
 |**hasError**|bool|optional|none|Boolean value indicating whether the column has an error in the data.|
 |**width**|number|optional|none|A number (in px) specifying the width of the column. If not provided, the Data Grid's default column width is used.|
+|**columnHighlightColor**|string|optional|none|The color to be used for highlighting a column.|
```

Comment:
The columnHighlightColor is an enumeration value.  The documentation should specify accordingly.  Also, the documentation does not give the default value.  Isn't the default value the orange color?

Reply:
Changed to enum: https://github.com/cerner/terra-framework/pull/1985/commits/8a2aac51a800bf755283f26bbdb29313b3703c04.
`columnHighlightColor` is no longer a direct prop under flowsheet, instead it would be part of column object.
Removed orange as default value. Now highlighting depends on the colour passed to each object (which in turn supports multi-highlight)

Reply:
We should definitely have a default value.

Reply:
I think we can avoid default value since each highlighted column represents a certain significance and will be set to a meaningful colour rather than always defaulting to orange. But if its strict requirement, we can make it default.